test/e2e-tests/README.md

```diff
@@ -0,0 +1,17 @@
+# AI Edge testing
+
+Place for testing and to verify the MLOps pipelines are working correctly. These tests use the `k8s.io/client-go` package to interact with the cluster. Using `oc login` to log into the cluster as normal should mean no setup is needed to run the tests.
+
+A locally install of the Go compiler is needed to run the tests. Go version `1.21` is required to run the tests.
```

Comment:
```suggestion
A local install of the Go compiler is needed to run the tests. Go version `1.21` is required to run the tests.
```

Reply:
Per this comment, is Go 1.21 what we should be using? It looks like we've configured Prow to use 1.19: https://github.com/openshift/release/blob/master/ci-operator/config/opendatahub-io/ai-edge/opendatahub-io-ai-edge-main.yaml#L5

@LaVLaS ^^

Reply:
We can definitely update the openshift-ci version if it causes issues.  I assume we just pinned the current release to 1.21 since it was the recent version and I don't think there are major changes that would affect running go test in go <= v1.19

Reply:
@jackdelahunt Everything looks good.  The suggestion above is the last [minor] issue to resolve and we can merge

Reply:
👍